napari/components/dims.py

```diff
@@ -340,6 +353,51 @@ def set_axis_label(
                     full_axis_labels[ax] = val
                 self.axis_labels = full_axis_labels
 
+    def set_thickness(
+        self,
+        axis: Union[int, Sequence[int]],
+        value: Union[Union[int, float], Sequence[Union[int, float]]],
+    ):
+        """Set the slider slice thickness for this dimension.
+
+        Parameters
+        ----------
+        axis : int or sequence of int
+            Dimension index or a sequence of axes whose slice thickness will be set.
+        value : scalar or sequence of scalars
+            Value of the slice thickness.
+        """
+        # * 2 allows to see the whole dim in one slice
```

Comment:
Is thickness then a half-width? i.e. slice is `(current - thickness, current + thickness)` or `(current - thickness / 2, current + thickness / 2)`? I would expect the latter - but then don't see why you need the `* 2` here

Reply:
It's the second. This `* 2` is not applied to the thickness but to the `max_thickness`. This might be wrong, but I thought that the most reasonable hard limit for slice thickness would be a slice that contains the whole dimension extent, no matter where it's centered (even if point is at an extreme, you still see the whole thing). This is unlikely to be actually useful, but it seemed an appropriate hard limit.

Reply:
oh ok, yeah - doesn't sound like `max_thickness` is so critical. we can leave this thread open though in case anyone else feels strongly

Reply:
yeah, this seems fine to me